python/lsst/jointcal/jointcal.py

```diff
@@ -357,6 +363,13 @@ def setDefaults(self):
         self.sourceSelector['science'].flags.bad = badFlags
 
 
+def writeModel(model, filename, log):
+    """Write model to outfile."""
+    with open(filename, "w") as file:
+        file.write(model.print())
+    log.info("Wrote model %s to file: %s", model, filename)
```

Comment:
This does leave a critical question: should the python interface have `str`->"ConstrainedAstrometryModel" and `repr`->"huge data dump" as implemented here, or should I leave `str/repr` as the defaults and implement the "big data dump" here as `print()` or `toString()` in python to prevent a user from accidentally getting the firehose? `repr` means it's easy to discover, which is both good and bad (and also means I need to implement a `str` version at the python level).

Reply:
Well, the former would be consistent with general Python convention.